sdks/java/extensions/euphoria/src/main/java/org/apache/beam/sdk/extensions/euphoria/core/translate/EuphoriaOptions.java

```diff
@@ -60,4 +70,13 @@ public TranslatorProvider create(PipelineOptions options) {
   AccumulatorProvider.Factory getAccumulatorProviderFactory();
 
   void setAccumulatorProviderFactory(AccumulatorProvider.Factory accumulatorProviderFactory);
+
+  @Description("Map of known PCollectionViews")
+  @Default.InstanceFactory(DefaultPCollectionViewStoreFactory.class)
+  @JsonIgnore
+  @Internal
+  PViewsStore getPCollectionViewsStore();
+
+  @Internal
+  void setPCollectionViewsStore(PViewsStore pCollectionViewsStore);
```

Comment:
is setter necessary?

Reply:
I reckon we don't need a separate field in pipeline options. It should be enough to keep instance property in BHJ Translator.

Reply:
Yes, I agree. But Gradle complains during build that there is a missing setter. Is there a way of circumventing it?
The `PViewsStore` will be skipped completely. As a conclusion of mutual discussion.